library/src/main/java/org/apache/apex/malhar/lib/state/managed/Bucket.java

```diff
@@ -232,6 +232,12 @@ public int hashCode()
 
     protected ConcurrentLinkedQueue<Long> windowsForFreeMemory = new ConcurrentLinkedQueue<>();
 
+    private static boolean disableBloomFilterByDefault = false;
+    private boolean disableBloomFilter = disableBloomFilterByDefault;
+    private static int bloomFilterDefaultBitSize = 1000000;
```

Comment:
In most of case, disableBloomFilter and bloomFilterBitSize should not be per bucket. It's kind depend the type of bucket. If this is the case, the static attributes can be used.
What I concern about put them into state manager is the it totally depends on Bucket implementation and not related to state manager at all. Let's think about we have another implementation, such as MemoryBucket, these attributes are non-sense to this implementation at all.